packages/procaptcha-react/src/components/CaptchaComponent.tsx

```diff
@@ -28,135 +28,143 @@ export interface CaptchaComponentProps {
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+                // fill entire screen
+                width: '100%',
+                height: '100%',
+            }}
+        >
             <Box
                 sx={{
-                    // center the popup horizontally and vertically
-                    display: 'flex',
-                    alignItems: 'center',
-                    justifyContent: 'center',
-                    // fill entire screen
+                    // introduce scroll bars when screen < minWidth of children
+                    overflowX: 'auto',
+                    overflowY: 'auto',
                     width: '100%',
-                    height: '100%',
+                    // limit the popup width
+                    maxWidth: '450px',
+                    // maxHeight introduces vertical scroll bars if children content longer than window
+                    maxHeight: '100%',
                 }}
             >
                 <Box
+                    bgcolor={theme.palette.background.default}
                     sx={{
-                        // introduce scroll bars when screen < minWidth of children
-                        overflowX: 'auto',
-                        overflowY: 'auto',
-                        width: '100%',
-                        // limit the popup width
-                        maxWidth: '450px',
-                        // maxHeight introduces vertical scroll bars if children content longer than window
-                        maxHeight: '100%',
+                        display: 'flex',
+                        flexDirection: 'column',
+                        // the min width of the popup before scroll bars appear
+                        minWidth: '300px',
                     }}
                 >
                     <Box
-                        bgcolor={theme.palette.background.default}
+                        px={2}
+                        py={3}
                         sx={{
+                            // center the header
                             display: 'flex',
-                            flexDirection: 'column',
-                            // the min width of the popup before scroll bars appear
-                            minWidth: '300px',
+                            alignItems: 'center',
+                            width: '100%',
                         }}
+                        bgcolor={theme.palette.primary.main}
                     >
-                        <Box
-                            px={2}
-                            py={3}
+                        <Typography
                             sx={{
-                                // center the header
-                                display: 'flex',
-                                alignItems: 'center',
-                                width: '100%',
+                                color: '#ffffff',
```

Comment:
does this not need to be set based on the theme?

Reply:
No, this is the topbar for the expanded captcha which is blue background white text